apps/condo/domains/banking/tasks/generateReports.js

```diff
@@ -149,10 +149,24 @@ const generateReports = async (taskId) => {
         deletedAt: null,
     })
 
+    if (!bankAccount) {
+        throw new Error(`Cannot find BankAccount by id="${bankAccountId}"`)
```

Comment:
shouldn't taskId be present here? Or will it already be in logs in case of `throw` ?

Reply:
Yes, this data is already in the logs. You can find it like this: `task.data.args`